src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
+  static alive = [];
```

Comment:
The `alive` static property is shared across all instances of `Animal`, `Herbivore`, and `Carnivore`. This could lead to unexpected behavior if different types of animals are not meant to be stored together. Consider whether this design is intentional or if separate arrays for different types of animals would be more appropriate.